### ISSUE TITLE
How do you use the nnUnet in your code and confuse about your architecture?

### ISSUE THREAD

**Issue by tphankr** [multimodallearning/convexAdam#2]
I saw in your paper at the feature extractor part is MIND and/or nnUnet. The MIND or nnUnet, are they model, right?
1) The MIND , did you use at the lines 232 and 233, right? So now, I am trying to do the nnUnet model, will I replace losses.MINDSSC by nnUnet, right? Are my mind right?, because I installed this model by pip install nnunet (https://github.com/MIC-DKFZ/nnUNet)  but do know import? 
![image](https://user-images.githubusercontent.com/37217354/146985193-5c502a5f-3502-4857-9ec0-5f1fa5e70ea5.png)

In short, I am wondering where is your model in your code?. Because your method training time is extremely fast and do not have checkpoints. Normorly, I know orthers method training time is about 20 hours or more.

Thank you @mattiaspaul.

**Comment by chrgrbr** [comment]
Hi @tphankr,
We have updated the codebase to make it a bit more readable. Yes, we rely on pre-computed features. Please have a look at our extended Journal publication!
